### PR TITLE
Keep focus on single click and update to use 1.6stable pending items api

### PR DESCRIPTION
**Pull request by kuychaco**
cc @BinaryMuse and @lee-dohm

### CODE REVIEW

File: lib/project/results-view.coffee [hunk 1]
```diff
@@ -21,13 +21,10 @@ class ResultsView extends ScrollView
       view.addClass('selected')
       if not e.ctrlKey
         if e.originalEvent?.detail is 1
-          @editorPromise = view.confirm(pending: true)
+          if view.hasClass('path') or atom.config.get('find-and-replace.openProjectFindResultsInRightPane')
```
Comment (BinaryMuse) [inline lib/project/results-view.coffee]: Which types of entries have the `path` class? Is this worth extracting into a helper for clarity?

**Comment by BinaryMuse** [comment]
Otherwise 👍

**Comment by Alhadis** [comment]
I loved being able to open stuff with a single click though. :(

**Comment by BinaryMuse** [comment]
@Alhadis That behavior was reverted in https://github.com/atom/find-and-replace/pull/704

**Comment by Alhadis** [comment]
Any chance there could be a user option to re-enable this?

I'm happy to submit a PR.

**Pull request merged**